python/coinbase-agentkit/coinbase_agentkit/agentkit.py

```diff
@@ -18,14 +18,9 @@ class AgentKitOptions(BaseModel):
 class AgentKit:
     """Main AgentKit class for managing wallet and action providers."""
 
-    def __init__(self, options: AgentKitOptions):
-        self.wallet_provider = options.wallet_provider
-        self.action_providers = options.action_providers or []
-
-    @classmethod
-    def from_options(cls, options: AgentKitOptions) -> "AgentKit":
-        """Create an AgentKit instance from options."""
-        return cls(options)
+    def __init__(self, config: AgentKitConfig):
+        self.wallet_provider = config.wallet_provider
+        self.action_providers = config.action_providers or []
```

Comment:
set default wallet provider to be `CdpWalletProvider` with API key from `AgentKitConfig` like in TS. and set default action provider to be `wallet_action_provider()`